### ISSUE TITLE
[BUG] v17.0.0 Fails to compile with yarn pnp

### ISSUE THREAD

**Issue by CEOehis** [microsoft/applicationinsights-react-js#56]
**Description**

Failure to compile with yarn pnp:

```sh
Failed to compile.

Module not found: Error: Can't resolve '@nevware21/ts-utils' in 'workspace/.yarn/__virtual__/@microsoft-applicationinsights-react-js-virtual-5e2603ec10/0/cache/@microsoft-applicationinsights-react-js-npm-17.0.0-f9df6f3087-3af1cfa49c.zip/node_modules/@microsoft/applicationinsights-react-js/dist-esm'
ERROR in ./.yarn/__virtual__/@microsoft-applicationinsights-react-js-virtual-5e2603ec10/0/cache/@microsoft-applicationinsights-react-js-npm-17.0.0-f9df6f3087-3af1cfa49c.zip/node_modules/@microsoft/applicationinsights-react-js/dist-esm/ReactPlugin.js 13:0-52
Module not found: Error: Can't resolve '@nevware21/ts-utils' in 'workspace/.yarn/__virtual__/@microsoft-applicationinsights-react-js-virtual-5e2603ec10/0/cache/@microsoft-applicationinsights-react-js-npm-17.0.0-f9df6f3087-3af1cfa49c.zip/node_modules/@microsoft/applicationinsights-react-js/dist-esm'
```

**Steps to Reproduce**
Minimal reproduction repo: https://github.com/CEOehis/application-insights-yarn-pnp

 - React Version: 17.0.2
 - SDK Versions:    
 	-	@microsoft/applicationinsights-react-js 17.0.0
 	-	@microsoft/applicationinsights-web 3.0.2

**Additional context**
`@nevware21/ts-utils` is [referenced in the code](https://github.com/microsoft/applicationinsights-react-js/blob/22b3e33d0196caeaf69c8b29672f5528b505338b/applicationinsights-react-js/src/ReactPlugin.ts#L16) but is not specified as a dependency . As a result, yarn pnp is unable to resolve it correctly.

It works with node-modules because `@nevware21/ts-utils` is also installed with `@microsoft/applicationinsights-web`.

**Comment by MSNev** [comment]
@siyuniu-ms, can you please look at this and add the referenced (and missing) dependencies, if we are using `ts-async` then we should also add that.

And we will create a release with this fix once we release v3.0.3 of ApplicationInsights (hopefully) next week.

**Comment by MSNev** [comment]
@siyuniu-ms, I've approved the PR. Can you please make sure that I've not missed something and if it looks good feel free to commit it.

**Comment by siyuniu-ms** [comment]
fixed at #57

**Comment by CEOehis** [comment]
@siyuniu-ms this doesn't seem to be fixed in #57.

I've updated my project to use @microsoft/applicationinsights-react-js 17.0.1 and @microsoft/applicationinsights-web 3.0.3 but I  still have the same error with yarn pnp.

It seems to me that the issue is that `@nevware21/ts-utils` is still not specified in the package.json as a dependency in this latest release.

**Comment by MSNev** [comment]
@siyuniu-ms can you please ensure that any used packages are included in the package.json (under dependencies) and if necessary fix and then publish an updated version
